document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/kafka/configurationHelper/impl/KafkaConfigHelperImpl.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.companieshouse.document.generator.core.kafka.configurationHelper.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.companieshouse.document.generator.core.kafka.configurationHelper.KafkaConfigHelper;
+import uk.gov.companieshouse.document.generator.core.kafka.configurationHelper.KafkaConsumerProducerConfigHelper;
+import uk.gov.companieshouse.kafka.consumer.ConsumerConfig;
+import uk.gov.companieshouse.kafka.producer.Acks;
+import uk.gov.companieshouse.kafka.producer.ProducerConfig;
+
+import java.util.List;
+
+public class KafkaConfigHelperImpl implements KafkaConfigHelper {
+
+    @Autowired
+    private KafkaConsumerProducerConfigHelper consumerProducerConfigHelperHelper;
```

Comment:
Typo on "consumerProducerConfigHelperHelper". You could shorten to "brokerConfigHelper".